bee-message/src/output/unlock_condition/mod.rs

```diff
@@ -101,29 +115,21 @@ impl TryFrom<Vec<UnlockCondition>> for UnlockConditions {
 
 impl UnlockConditions {
     ///
-    pub const COUNT_MAX: u8 = 3;
+    pub const COUNT_MAX: u8 = 6;
```

Comment:
Do tests acknowledge this change?

Reply:
No, I'm waiting a bit before writing all the tests, TIP change too fast. I'll do a PR adding them when it's stable enough.